cmd/service.go

```diff
@@ -102,7 +102,12 @@ func (m *ServerMux) handleServiceSignals() error {
 			}
 			runExitFn(nil)
 		case serviceStop:
-			log.Println("Gracefully stopping... (press Ctrl+C again to force)")
+			// Extra space will prevent showing "^CExiting" and rather show "^C Exiting" instead.
+			log.Println(" Exiting")
```

Comment:
`Received exit signal (Ctrl+C/SIGTERM/etc)! Attempting to gracefully exit.` is too long and verbose.
`Received signal to exit.` will be better